src/containers/projectcontainer.js

```diff
@@ -52,6 +52,21 @@ class ProjectContainer extends Component {
     this.props.updateField(fieldName, value);
   }
 
+  generateProjectId = (customerId) => {
+    const prefix = this.props.customers.data.get(customerId).name
+      .substring(0, 3)
+      .toUpperCase();
+
+    const nextId = this.props.projects.data.toSeq()
+    .filter(project => project.customer.id === customerId)
+    .reduce((previousId, currentProject) => {
+      const currentNumber = parseInt(currentProject.id.replace(/.*\D/g, ''));
```

Comment:
I used a regex tool, but this lacks testing. With your example it concatenates all digits, with the current one, it only picks consecutive digits (ideally)